src/pocketmine/block/Cactus.php

```diff
@@ -127,4 +129,18 @@ public function place(Item $item, Block $blockReplace, Block $blockClicked, int
 
 		return false;
 	}
+
+	public function canBlockLand(Fallable $block) : int{
+		$parentDecision = parent::canBlockLand($block);
+		if ($parentDecision == FallingBlock::STATE_DROP_ITEM) {
+			return $parentDecision;
+		}
+
+		if ($block instanceof Sand) {
```

Comment:
what about concrete?

Reply:
is there a list of blocks which has the same behaviour? does it work for all falling blocks?

Reply:
As far as I know, yes.